app/Http/Controllers/UpdatePersonalInfo.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\UpdatePersonalInfoRequest;
+use App\Transformers\UserTransformer;
+use Illuminate\Contracts\Routing\ResponseFactory;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Response;
+
+class UpdatePersonalInfo extends Controller
+{
+    public function __invoke(UpdatePersonalInfoRequest $request): JsonResponse|ResponseFactory
+    {
+        $user = $request->user();
+        $user->update($request->validated());
```

Comment:
It would be nice to have empty lines separating lines 17, 18, and 19 so that it looks cleaner.

Reply:
Done